### ISSUE TITLE
UNINSTALL???

### ISSUE THREAD

**Issue by ercan3582** [Rem0o/FanControl.Releases#933]
After installing Fan Control some of my system fans are not detected and no longer work. I'm trying to uninstall the software, but it does not have an uninstall.exe, and it does not show up in add/remove programs on windows 10. Please help with a solution for removing this software, thanks.

**Comment by Rem0o** [comment]
There is nothing to "uninstall".  It's just an executable.

**Comment by ercan3582** [comment]
Sorry for my lack of computer knowledge.. but when I try to delete the folder that had the .exe (the zip folder) it deletes everything except for "FanControl.sys" and my fans still aren't working. How do I remove any traces of this software so my pc works properly

**Comment by Rem0o** [comment]
That means sys file is still active, probably because it failed to exit properly. Restarting your computer should allow you to delete it.

**Comment by FrancoisMentec** [comment]
Hello, same issue, I rebooted my computer yet I still cannot delete FanControl.sys and I have a popup on system startup that tells me it did not find FanControl (which is expected since I deleted it). I cannot find anything and Windows startup programs or services.

**Comment by Rem0o** [comment]
@FrancoisMentec lookup task scheduler.

**Comment by dedood2008** [comment]
lookup task scheduler. and  than  what?

**Comment by Rem0o** [comment]
@dedood2008 there will be task named "FanControl" if you checked "Start with windows". Just delete it.

**Comment by kn04nd** [comment]
I have installed the FanControl using winget, then, I've noticed that do not works with my laptop Alienware m15 r3, so then I tryied to unisntall but doesn't work, so, i tried to delete de C:\Program Files (x86)\FanControl folder, no sucess, deleting file by file, only the file fancontrol.sys was not possible to delete, I look up in processes and do not have anything, also looked in tash scheduler and do have nothing in there about fancontrol, tried to restart my laptop, and again, no success when deleting the damm file.
And now? what can I do?
